commit e66e1f6528cbae553338793e0eae4125033369b6 (refs/changes/80/5380/3)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-01-17T18:59:05+00:00 (9 months ago)
    
    TRITON-633 dangling symlinks break the adminui build when using /usr/bin/cp

diff --git a/www/img/glyphicons-halflings-white.png b/www/img/glyphicons-halflings-white.png
deleted file mode 120000
index 08fa0f2b..00000000
--- a/www/img/glyphicons-halflings-white.png
+++ /dev/null
@@ -1 +0,0 @@
-../../bootstrap/img/glyphicons-halflings-white.png
\ No newline at end of file
diff --git a/www/img/glyphicons-halflings.png b/www/img/glyphicons-halflings.png
deleted file mode 120000
index 10ff0466..00000000
--- a/www/img/glyphicons-halflings.png
+++ /dev/null
@@ -1 +0,0 @@
-../../bootstrap/img/glyphicons-halflings.png
\ No newline at end of file
